### PR TITLE
Style touchups

### PR DESCRIPTION
**Pull request by janegreene**
Style touch ups to nav bar, welcome, about, podcast show, and README

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1,15 +1,15 @@
 <% if current_user %>
 
 <section class="podcast_following center-search">
-  <h1><%= @user.following_podcasts.length %> Following</h1>
+  <h1>Shows you are Following: <%= @user.following_podcasts.length %></h1>
   <% @user.following_podcasts.each do |podcast| %>
     <section id="follow">
-      <%= link_to podcast.name, "/podcasts/#{podcast.id}" %>
+      <%= button_to  podcast.name , podcast_path(podcast.id), method: 'get', class: "btn btn-outline btn-xl js-scroll-trigger" %>
```
Comment (janegreene) [inline app/views/users/show.html.erb:7]: I made this the normal button we have been using and changed the tests to match.

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -127,7 +127,7 @@ section h2 {
 @media (min-width: 992px) {
   #mainNav {
     border-color: transparent;
-    background-color: transparent;
+    background-color: #4e4a4aa6;
```
Comment (kiverso) [inline app/assets/stylesheets/application.scss:130]: Looks much better but is still semi transparent. Do we want to make it completely solid so there are no conflicts with text in the nav and text in the page?

File: app/views/sessions/new.html.erb [hunk 3]
```diff
@@ -1,18 +1,4 @@
 <div class='center-search'>
   <p>Please, login using Google.</p>
+<a class="btn btn-outline btn-xl js-scroll-trigger" href='/auth/google_oauth2'>Login With Google</a>
```
Comment (kiverso) [inline app/views/sessions/new.html.erb:3]: looks much cleaner, nice job.

File: app/views/users/show.html.erb (hunk 1 above)
Comment (kiverso) [inline app/views/users/show.html.erb:7]: the buttons are a great addition but there still feels like a lot of white space here. Possible something to look to clean up if we have time.

File: app/assets/stylesheets/application.scss (hunk 2 above)
Comment (janegreene) [reply app/assets/stylesheets/application.scss:130]: @muydanny can decide, he wasn't a fan of the stronger opacity I had tried behind the podcast description.

Comment (kiverso) [reply app/assets/stylesheets/application.scss:130]: okay, sounds good to keep it as is for now then.

File: app/views/users/show.html.erb (hunk 1 above)
Comment (janegreene) [reply app/views/users/show.html.erb:7]: Maybe even a "future us" ticket for making linked cards of the thumbnails would be cool.

**Pull request merged**